### PR TITLE
Add *Scan blog post

### PR DESCRIPTION
**Pull request by AntonBikineev**

### CODE REVIEW

**Comment by mlippautz** [comment]
Can you make sure the SVGs are somewhat viewable with dark backgrounds (e.g., make their background transparent).

Did you run the SVGs through SVGOMG (https://jakearchibald.github.io/svgomg/)?

File: src/blog/retrofitting-temporal-memory-safety-on-c++.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+author: 'Anton Bikineev, Michael Lippautz ([@mlippautz](https://twitter.com/mlippautz)), Hannes Payer ([@PayerHannes](https://twitter.com/PayerHannes))'
+avatars:
+  - anton-bikineev
+  - michael-lippautz
+  - hannes-payer
+date: 2022-06-14
+tags:
+  - security
+  - scan
+description: 'V8 ships with Oilpan, a garbage collection library for hosting managed C++ memory.'
```
Comment (mlippautz) [inline src/blog/retrofitting-temporal-memory-safety-on-c++.md:12]: Stale description.

File: src/blog/retrofitting-temporal-memory-safety-on-c++.md [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+
+Over the last decade, another approach has seen some success: memory quarantine. The basic idea is to put explicitly freed memory into quarantine and only make it available when a certain safety condition is reached. Microsoft has shipped versions of this mitigation in its browsers: [MemoryProtector](https://www.trendmicro.com/en_us/research/14/g/mitigating-uaf-exploits-with-delay-free-for-internet-explorer.html) in Internet Explorer in 2014 and its successor [MemGC](https://securityintelligence.com/memgc-use-after-free-exploit-mitigation-in-edge-and-ie-on-windows-10/) in (pre-Chromium) Edge in 2015. In the [Linux kernel](https://a13xp0p0v.github.io/2020/11/30/slab-quarantine.html) a probabilistic approach was used where memory was eventually just recycled. And this approach has seen attention in academia in recent years with the [MarkUs paper](https://www.cst.cam.ac.uk/blog/tmj32/addressing-temporal-memory-safety). The rest of this article summarizes our journey of experimenting with quarantines and heap scanning in Chrome.
+
+(At this point, one may ask where pointer authentication fits into this picture – keep on reading!)
```
Comment (planglois) [inline src/blog/retrofitting-temporal-memory-safety-on-c++.md:36]: I didn't see any uses of pointer authentication, did you mean to say Memory Tagging here instead?

**Review by planglois — COMMENTED** [review]
Hi there,

I just noticed what looks like a typo while reading the blog post so I thought I'd leave a note. Interesting stuff!

Thanks,
Pierre

File: src/blog/retrofitting-temporal-memory-safety-on-c++.md (hunk 1 above)
Comment (AntonBikineev) [reply src/blog/retrofitting-temporal-memory-safety-on-c++.md]: Fixe

File: src/blog/retrofitting-temporal-memory-safety-on-c++.md (hunk 2 above)
Comment (mlippautz) [reply src/blog/retrofitting-temporal-memory-safety-on-c++.md:36]: I think when I wrote this I meant pointer authentication in a broad sense and not ARM's PAC mechanisms in particular.

After all, memory tagging also ensures pointer authenticity, it's just not cryptographically save. We cann call this memory tagging instead.

Comment (planglois) [reply src/blog/retrofitting-temporal-memory-safety-on-c++.md:36]: Oh I see, that makes sense. It  might be a little confusing for somebody familiar with PAC, so memory tagging might be clearer indeed, but it's not that important.

Comment (AntonBikineev) [reply src/blog/retrofitting-temporal-memory-safety-on-c++.md]: Changed to "memory tagging", thanks!

**Review by mlippautz — APPROVED** [review]

**Comment by mathiasbynens** [comment]
Do y’all still have the original docs/sheets/slides with the source for these images? I’m asking because then I could export SVG versions and replace these PNGs with them, so the images would look better on v8.dev.

**Pull request merged**